src/intTest/java/uk/nhs/digital/nhsconnect/nhais/uat/OutboundMeshServiceUAT.java

```diff
@@ -47,20 +56,27 @@ void testTranslatingFromFhirToEdifact(String category, TestData testData) throws
         // send EDIFACT to API
         sendToApi(testData.getFhir(), transactionType);
 
-        // fetch EDIFACT from "outbound queue"
-        var gpOutboundQueueMessage = getOutboundQueueMessage();
+        // fetch EDIFACT message from MESH
+        await().atMost(10, TimeUnit.SECONDS)
+            .pollDelay(Durations.FIVE_SECONDS)
```

Comment:
Would the tests run faster if we polled at a higher frequency?

Reply:
Using this:
````java
        await().atMost(10, TimeUnit.SECONDS)
            .pollInterval(Durations.TWO_HUNDRED_MILLISECONDS)
            .until(this::isNewMessageAvailable);
````
one test takes ~6,5 seconds, when above one took 12 sec

Reply:
I've tried 5sec/5sec and sometimes tests are failing

Reply:
the problem is you used `.pollDelay()` instead of `.pollInterval()`